src/components/Lightbox.tsx

```diff
@@ -31,10 +28,10 @@ type LightboxProps = {
     isAuthTokenRequired: boolean;
 
     /** URI to full-sized attachment, SVG function, or numeric static image on native platforms */
-    source: ImageSourcePropType;
+    uri: string;
 
     /** Triggers whenever the zoom scale changes */
-    onScaleChanged: OnScaleChangedCallback;
+    onScaleChanged?: OnScaleChangedCallback;
```

Comment:
`onScaleChanged` isn't crucial for the `Lightbox` component to work. I implemented `Lightbox` and `MultiGestureCanvas` as generically as possible, so that we can easily use it in other uses cases in the future...